### PR TITLE
Add javadocs in the QueuedStatementResource methods

### PR DESCRIPTION
**Pull request by andreyliporage**
# Pull Request Merge Checklist
           
Task backlog identifier related to this PR: ---

---

## As a DEVELOPER, I confirm that this PR **doesn't contain**:
- [x] Code lines with syntax errors
- [x] Methods and variables names that aren't clear or doesn't mean anything
- [x] Code that is out of the standard (Example: **PEP-8** for Python code)
- [x] Files with **imports** errors or without any utility
- [x] **TODOs** left behind
- [x] Tests that are being skipped
- [x] Methods placed on incorrect files or folders (Example: methods that are local, but should be placed on /utils)
- [x] Functions, methods or API responses that are publicly returning sensitive information

## As a DEVELOPER, I confirm that I:
- [x] Followed the Simbiose's standard for branch and commit naming
- [x] Concluded the task and achieved the goals according to the task planning I defined before starting to code
- [x] Added all necessary database changes to the [SQL Migrations](https://github.com/s1mbi0se/sql-migrations)
- [x] Created tests for every new flow and method on this Pull Request
- [x] Added the necessary DEBUG logs
- [x] Checked my code using the Simbiose's Corner Cases Guide
- [x] Merged the **develop** branch to this PR/branch before sending the PR for review
- [x] Solved all possible merge conflicts before sending the PR for review
- [x] Corrected all the problems and code smells identified by the quality tools
- [x] Manually executed the necessary tests locally to make sure my changes didn't break anything
- [x] All unitary and integration tests ran by the CI system are passing
- [x] Check the performance of my changes or additions and they didn't make the system slower (I profiled the code)
- [x] Updated the necessary documentation (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Made the self-review of the code before sending the PR for review
- [x] Sent this PR for a peer-review
- [x] Made all the corrections appointed during the peer-review
- [x] Confirmed that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block my merge

---  

## As the TEAM MANAGER, I confirm that I:
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked if the purpose of the task was reached with this PR
- [x] Checked that the code was developed following the best practices for the language, with the best possible readability and using the less possible quantity of code lines and complexity. 
- [x] Checked if all related PRs were created when other components were affected by this task (e.g. database changes on [SQL Migrations](https://github.com/s1mbi0se/sql-migrations))
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s pattern for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

---  

## As the QUALITY RESPONSIBLE, I confirm that I:
- [x] Checked if the Pull Request and commit messages follow the Simbiose’s defined standard 
- [x] Checked if there is no possible merge conflicts before reviewing the PR
- [x] Checked that were created unit, integration and chaos tests for every new flow and method on this Pull Request
- [x] Checked that no sensible information (e.g. users security data, internal security data, etc) was exposed by this PR
- [x] Checked that code coverage for this PR is higher than 80%.
- [x] Checked that there are no problems and code smells identified by the quality and security tools
- [x] Checked that all files were created on the proper folders according to the Simbiose’s defined standard
- [x] Checked that all classes, methods and variables on this PR follow the Simbiose’s code standard for this programming language
- [x] Checked that all unitary and integration tests ran by the CI system are passing
- [x] Checked that there are no performance issues created by these changes or additions
- [x] Checked that all documentation is up to date according to this change (Example: OpenAPI, Docstrings, Metadata Database Modeling)
- [x] Checked that this PR merge will not cause other systems or dependencies to stop working (Example: ShannonDB Python Client and ShannonDB)
- [x] Checked all GitHub Actions related to this PR that can block the merge

## Important links:
1. [Simbiose Quality Guidelines](https://github.com/s1mbi0se/company/wiki/Quality-Topics#table-of-contents)
2. [Metadata Database Modeling](https://docs.google.com/spreadsheets/d/1Y8_k5xV86kTjgfjbi-PGsJcubUKhWlT2x8ecpnMlyR0/edit?usp=sharing)
3. [Simbiose's Corner Cases Guide](https://www.simbioseventures.com/documentos-e-guias/procedimentos-e-checklists/corner-cases-guide)

### CODE REVIEW

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 1]
```diff
@@ -52,21 +63,41 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
...
+     *
+     * @return an Option {@link CoordinatorLocation}.
+     */
     public Optional<CoordinatorLocation> getCoordinatorLocation()
     {
         return coordinatorLocation;
     }
 
+    /**
+     * Returns if there was a failure during the query.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:77]: if there was a failure, it returns detailed information such the type, the message and cause.

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 2]
```diff
@@ -52,21 +63,41 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
...
+     *
+     * @return a {@link Duration} object.
+     */
     public Duration getElapsedTime()
     {
         return elapsedTime;
     }
 
+    /**
+     * Returns the waiting time for a query.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:97]: returns the queued time. The waiting time does not make it clear that the query hasn't started yet.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 3]
```diff
@@ -157,6 +157,20 @@ public void stop()
         queryPurger.shutdownNow();
     }
 
+    /**
+     * Gets the information about the query.
+     * <p>
+     * Checks if there is a sql command, takes the remote address from the request, goes
+     * through the headers to instantiate an new object of type {@link HttpRequestSessionContext}
+     * and type {@link Query} and returns a response from the server.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:165]: The method actually receives statement requests which creates new queries on Presto. It doesn't get info about existing queries.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 4]
```diff
@@ -242,6 +256,17 @@ private Query getQuery(QueryId queryId, String slug, long token)
         return query;
     }
 
+    /**
+     * Takes the uri from the html query.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:260]: It actually builds a new Query HTML URI for the specified queryId.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 5]
```diff
@@ -250,6 +275,19 @@ private static URI getQueryHtmlUri(QueryId queryId, UriInfo uriInfo)
                 .build();
     }
 
+    /**
+     * Takes the uri from the queue.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:279]: It builds a new Queued URI for the specified queryId.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 6]
```diff
@@ -261,6 +299,19 @@ private static URI getQueuedUri(QueryId queryId, Slug slug, long token, UriInfo
                 .build();
     }
 
+    /**
+     * Instantiates an object of type {@link QueryResults} with information about
+     * the result of the query.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:304]: and also all query related metadata.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 7]
```diff
@@ -411,6 +462,20 @@ private QueryResults createQueryResults(long token, UriInfo uriInfo, DispatchInf
                     dispatchInfo.getQueuedTime());
         }
 
+        /**
+         * Checks if the query is completed or directs to another uri.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:466]:
```suggestion
         * Checks if the query is completed or redirects to another uri.
```

File: presto-main/src/main/java/io/prestosql/server/protocol/Slug.java [hunk 8]
```diff
@@ -30,6 +30,12 @@
 
     private static final SecureRandom RANDOM = new SecureRandom();
 
+    /**
+     * Checks if it is a thread safe and returns an instance a new object
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/server/protocol/Slug.java:34]: It instantiates a new Slug object

File: presto-main/src/main/java/io/prestosql/server/protocol/Slug.java [hunk 9]
```diff
@@ -44,6 +50,15 @@ private Slug(byte[] slugKey)
         this.hmac = Hashing.hmacSha1(requireNonNull(slugKey, "slugKey is null"));
     }
 
+    /**
+     * Computes a hash code based on the data that have been provided
+     * and turns it into a string
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/server/protocol/Slug.java:55]: string slug

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 10]
```diff
@@ -74,7 +74,8 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
     }
 
     /**
-     * Returns if there was a failure during the query.
+     * If there was a failure, it returns detailed information such the type, the message
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:77]:
```suggestion
     * If there was a failure, it returns detailed information such as the type, the message
```

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 11]
```diff
@@ -94,7 +95,7 @@ public Duration getElapsedTime()
     }
 
     /**
-     * Returns the waiting time for a query.
+     * Returns the wait time that the query has not yet started.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:98]: queued time

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 12]
```diff
@@ -160,9 +160,7 @@ public void stop()
     /**
      * Gets the information about the query.
      * <p>
-     * Checks if there is a sql command, takes the remote address from the request, goes
-     * through the headers to instantiate an new object of type {@link HttpRequestSessionContext}
-     * and type {@link Query} and returns a response from the server.
+     * Receives instruction requests that create new queries in the future.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:163]:
```suggestion
     * Receives instruction requests that create new queries on Presto.
```

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 13]
```diff
@@ -160,9 +160,7 @@ public void stop()
     /**
      * Gets the information about the query.
```
Comment (caiopavanelli) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:161]:
```suggestion
     * Creates new queries on Presto.
```

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 14]
```diff
@@ -27,6 +27,17 @@
     private final Duration elapsedTime;
     private final Duration queuedTime;
 
+    /**
+     * Takes the waiting time and the elapsed time.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:31]: This is not the purpose of the method. What it really does with these times?
What does this DispatchInfo means?

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 15]
```diff
@@ -27,6 +27,17 @@
...
+     * <p>
+     * Receives elapsed time and waiting time and return
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:33]: What is this time for?

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 16]
```diff
@@ -52,21 +63,42 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
         this.queuedTime = requireNonNull(queuedTime, "queuedTime is null");
     }
 
+    /**
+     * Returns the coordinator's uri.
+     *
+     * @return an Option {@link CoordinatorLocation}.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:69]: "an Optional {@link CoordinatorLocation} object"

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 17]
```diff
@@ -52,21 +63,42 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
...
+     */
     public Optional<CoordinatorLocation> getCoordinatorLocation()
     {
         return coordinatorLocation;
     }
 
+    /**
+     * If there was a failure, it returns detailed information such as the type, the message
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:77]: The summary should be only: "Returns detailed information about a given failure."

Your description could bring the details: "Returns detailed information about an executed failure, such as its type,  message and cause."

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java [hunk 18]
```diff
@@ -52,21 +63,42 @@ private DispatchInfo(Optional<CoordinatorLocation> coordinatorLocation, Optional
...
+     *
+     * @return a {@link Duration} object.
+     */
     public Duration getElapsedTime()
     {
         return elapsedTime;
     }
 
+    /**
+     * Returns the queued time.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchInfo.java:98]: I always need to know what this time is related to:
"Returns the queued time of the query."

File: presto-main/src/main/java/io/prestosql/execution/QueryIdGenerator.java [hunk 19]
```diff
@@ -107,6 +109,9 @@ public synchronized QueryId createNextQueryId()
         return new QueryId(format("%s_%05d_%s", lastTimestamp, counter++, coordinatorId));
     }
 
+    /**
+     * @return response time in milliseconds
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/execution/QueryIdGenerator.java:113]: Add a summary please: "Returns the actual time in milliseconds."

File: presto-main/src/main/java/io/prestosql/server/protocol/Slug.java [hunk 20]
```diff
@@ -30,6 +30,11 @@
 
     private static final SecureRandom RANDOM = new SecureRandom();
 
+    /**
+     * Instantiates a new Slug object.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/server/protocol/Slug.java:34]: I think it would be nice to explain in a description what this Slug object represents

File: presto-main/src/main/java/io/prestosql/dispatcher/DispatchManager.java [hunk 21]
```diff
@@ -129,6 +129,11 @@ public QueryManagerStats getStats()
         return stats;
     }
 
+    /**
+     * Generates next queryId.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/DispatchManager.java:133]: Generates the next query identifier.

File: presto-main/src/main/java/io/prestosql/server/protocol/Slug.java [hunk 22]
```diff
@@ -44,6 +49,15 @@ private Slug(byte[] slugKey)
...
     }
 
+    /**
+     * Computes a hash code based on the data that have been provided
+     * and turns it into a string slug.
+     *
+     * @param context fixed set of constants
+     * @param token generated token
+     *
+     * @return
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/server/protocol/Slug.java:59]: Remember to add a description

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 23]
```diff
@@ -157,6 +157,18 @@ public void stop()
         queryPurger.shutdownNow();
     }
 
+    /**
+     * Creates new queries on Presto.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:161]: Creates new queries to be executed on Presto.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 24]
```diff
@@ -157,6 +157,18 @@ public void stop()
...
+     * <p>
+     * Receives instruction requests that create new queries on Presto.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:163]: Receives POST requests that tries to create new queries to be executed on Presto.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 25]
```diff
@@ -157,6 +157,18 @@ public void stop()
...
+     *
+     * @param statement a sql command.
+     * @param servletRequest an object that provides request information for HTTP servlets.
+     * @param httpHeaders an object responsible for the http request headers.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:167]: an object containing the HTTP request headers

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 26]
```diff
@@ -157,6 +157,18 @@ public void stop()
...
+     * @param uriInfo an object responsible for taking information from the uri.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:168]: an object containing all extra information that should be added to the uri.
[Confirm with Caio]

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 27]
```diff
@@ -242,6 +254,17 @@ private Query getQuery(QueryId queryId, String slug, long token)
         return query;
     }
 
+    /**
+     * Builds a new QueryHTMLURI for the specified queryId.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:258]: Confirm what this method should be doing with Caio

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 28]
```diff
@@ -250,6 +273,19 @@ private static URI getQueryHtmlUri(QueryId queryId, UriInfo uriInfo)
                 .build();
     }
 
+    /**
+     * Builds a new QueuedURI for the specified queryId.
+     * <p>
+     * Replaces the existing path and add the query id at the end of the uri.
+     * Clear existing query parameters and then delegate to UriBuilder.query(String).
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:280]: Then, builds the new QueuedURI for the specified queryId.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 29]
```diff
@@ -261,6 +297,19 @@ private static URI getQueuedUri(QueryId queryId, Slug slug, long token, UriInfo
                 .build();
     }
 
+    /**
+     * Instantiates an object of type {@link QueryResults} with information about
+     * the result of the query and also all query related metadata.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:302]: Define that only query metadata should be returned

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 30]
```diff
@@ -261,6 +297,21 @@ private static URI getQueuedUri(QueryId queryId, Slug slug, long token, UriInfo
                 .build();
     }
 
+    /**
+     *
+     * <p>
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:302]: Where is the summary? Should remove the paragraph?

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 31]
```diff
@@ -261,6 +297,21 @@ private static URI getQueuedUri(QueryId queryId, Slug slug, long token, UriInfo
...
+     * Instantiates an object of type {@link QueryResults} with information about
+     * all query related metadata.
+     *
+     * @param queryId an object responsible for the query id.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:306]: an object containing the query identifier.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 32]
```diff
@@ -411,6 +462,20 @@ private QueryResults createQueryResults(long token, UriInfo uriInfo, DispatchInf
                     dispatchInfo.getQueuedTime());
         }
 
+        /**
+         * Checks if the query is completed or redirects to another uri.
+         * <p>
+         * Checks if the error message fails, if it fails it is because the query was completed.
+         * If it doesn't fail, it takes the coordinator's location and redirects the query to another
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:469]: If not ....

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 33]
```diff
@@ -411,6 +462,20 @@ private QueryResults createQueryResults(long token, UriInfo uriInfo, DispatchInf
...
+         * uri.
+         *
+         * @param token generated token
+         * @param uriInfo an object responsible for taking metadata from the uri.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:473]: uriInfo an object containing the uri metadata.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 34]
```diff
@@ -157,6 +157,18 @@ public void stop()
...
 
+    /**
+     * Creates new queries to be executed on Presto.
+     * <p>
+     * Receives POST requests that tries to create new queries to be executed on Presto.
+     *
+     * @param statement a sql command.
+     * @param servletRequest an object that provides request information for HTTP servlets.
+     * @param httpHeaders an object containing the HTTP request headers.
+     * @param uriInfo an object responsible for taking metadata from the uri.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:168]: an object containing the uri metadata.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 35]
```diff
@@ -242,6 +254,17 @@ private Query getQuery(QueryId queryId, String slug, long token)
         return query;
     }
 
+    /**
+     * Builds the uri that leads to the query.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:258]: Builds the specific query uri.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 36]
```diff
@@ -242,6 +254,17 @@ private Query getQuery(QueryId queryId, String slug, long token)
...
+     * <p>
+     * Determines the default path where the query took place and the parameters
+     * passed by the query.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:261]: passed = defined

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 37]
```diff
@@ -242,6 +254,17 @@ private Query getQuery(QueryId queryId, String slug, long token)
...
+     *
+     * @param queryId an object responsible for the query id.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:263]: an object containing the query id

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 38]
```diff
@@ -242,6 +254,17 @@ private Query getQuery(QueryId queryId, String slug, long token)
...
+     * @param uriInfo an object responsible for taking metadata from the uri.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:264]: an object containing the uri metadata.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 39]
```diff
@@ -242,6 +254,17 @@ private Query getQuery(QueryId queryId, String slug, long token)
...
+     *
+     * @return a {@link URI} object.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:266]: an

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 40]
```diff
@@ -250,6 +273,19 @@ private static URI getQueryHtmlUri(QueryId queryId, UriInfo uriInfo)
...
+    /**
+     * Builds a new QueuedURI for the specified queryId.
+     * <p>
+     * Replaces the existing path and add the query id at the end of the uri.
+     * Then, builds the new QueuedURI for the specified queryId.
+     *
+     * @param queryId an object responsible for the query id.
+     * @param slug an object responsible for translating a multibyte value into bytes.
+     * @param token generated token.
+     * @param uriInfo an object responsible for taking metadata from the uri.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:285]: an object containing the uri metadata.

File: presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java [hunk 41]
```diff
@@ -250,6 +273,19 @@ private static URI getQueryHtmlUri(QueryId queryId, UriInfo uriInfo)
                 .build();
     }
 
+    /**
+     * Builds a new QueuedURI for the specified queryId.
+     * <p>
+     * Replaces the existing path and add the query id at the end of the uri.
+     * Then, builds the new QueuedURI for the specified queryId.
+     *
+     * @param queryId an object responsible for the query id.
```
Comment (joaopedro-simbiose) [inline presto-main/src/main/java/io/prestosql/dispatcher/QueuedStatementResource.java:282]: an object containing the query id.
